### PR TITLE
Fix height of Main map

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]
doesn't work
image
image

it should be fixed in another way

**Review by 6mahdapihka9 — APPROVED** [review]
seems good to me

File: frontend/src/components/Map/index.tsx [hunk 1]
```diff
@@ -45,11 +45,11 @@ function Map({ onOpenBigPopup }: Props) {
   // }
 
   return (
-    <Box>
+    <Box sx={{ height: '100%' }}>
       <MapContainer
         center={[50.447731, 30.542721]}
         zoom={6}
-        style={{ height: '100vh' }}
+        style={{ height: '100%' }}
```
Comment (steadyGuy) [inline frontend/src/components/Map/index.tsx:52]: why style? sx should work as well

Comment (steadyGuy) [reply frontend/src/components/Map/index.tsx:52]: okay
I see
MapContainer is from leaflet.

**Review by steadyGuy — APPROVED** [review]

**Review by romawht22 — APPROVED** [review]

**Pull request merged**